javascript/code-challenges/challenges-03.test.js

```diff
@@ -0,0 +1,312 @@
+'use strict';
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 1 - Review
+
+Write a function called addTwo that takes in an array and adds two to every value using a for loop. Place the new value in a new array. Return the new array.
+------------------------------------------------------------------------------------------------ */
+
+const addTwo = (arr) => {
+  // Solution code here...
+  let newArr = [];
+
+  for (let i = 0; i < arr.length; i++){
+    let result =arr[i] + 2;
+    newArr.push(result);
+  }
+  return newArr;
+};
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 2
+
+Write a function named typeNum that, given an array as input, uses filter to return an array containing only the numbers.
+
+For example, typeNum([1, 'bob' ,3]) returns [1,3].
+------------------------------------------------------------------------------------------------ */
+
+const typeNum = (arr) => {
+  // Solution code here...
+  let newArr = arr.filter(num => typeof(num)=== 'number');
+  return newArr;
+};
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 3
+
+Write a function named containsAnd that, given an array of strings as input, uses filter to return an array containing only strings that contain 'and' within the string.
+
+For example, containsAnd(['panda', 'ran', 'and']) returns ['panda', 'and'].
+------------------------------------------------------------------------------------------------ */
+
+const containsAnd = (arr) => {
+  // Solution code here...
+  let newArr = arr.filter(num => num.includes('and'));
+  return newArr;
+};
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 4
+
+Write a function named oddValues that, given an array of integers as input, uses filter to return an array containing only the odd integers.
+
+For example, oddValues([1,2,3]) returns [1,3].
+------------------------------------------------------------------------------------------------ */
+
+const oddValues = (arr) => {
+  // Solution code here...
+  let newArr = arr.filter(num => num % 2 !== 0);
+  return newArr;
+};
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 5
+
+Write a function named notInFirstArray that, given two arrays as input, uses filter to return an array of all the elements in the second array that are not included in the first array.
+
+For example, notInFirstArray([1,2,3], [1,2,3,4]) returns [4].
+------------------------------------------------------------------------------------------------ */
+
+const notInFirstArray = (forbiddenValues, arr) => {
+  // Solution code here...
+  let newArr = forbiddenValues.filter(numb => !arr.includes(numb));
+  return newArr;
+};
```

Comment:
Hi there, great job on these! 👍 

You are close with this one, we just need to switch some things around. 
- Is it the `forbiddenValues` we want to run filter() on or perhaps the `arr`?